pkgs/development/python-modules/coapthon3/default.nix

```
@@ -0,0 +1,20 @@
+{ lib, pythonPackages }:
+
+pythonPackages.buildPythonPackage (with pythonPackages; rec {
```

```suggestion
{ lib, pythonPackages, cachetools }:

pythonPackages.buildPythonPackage rec {
```

This is usually not done because I think it breaks the compatibility with other python versions. The correct way is to pass in the other python modules directly.

---

Done. I did the same with `fetchPypi` and `buildPythonPackages`. Also removed the `)`.